contracts/test/PDPService.t.sol

```diff
@@ -7,14 +7,16 @@ import {Cids} from "../src/Cids.sol";
 
 contract PDPServiceProofSetCreateDeleteTest is Test {
     PDPService pdpService;
+    PDPRecordKeeper recordKeeper;
 
     function setUp() public {
         pdpService = new PDPService(2);
+        recordKeeper = new PDPRecordKeeper(address(pdpService));
```

Comment:
As I'll be adding assertions on the state of the recordkeeper at the end of each test to finish up this PR it is somewhat the case that PDPService tests are becoming too coupled to the PDPRecordKeeper contract.  However my thinking is that 
* The current PDPRecordKeeper which keeps everything matches perfectly with how I would implement fake record keeper
* If we end up tweaking PDPRecordKeeper to be smarter and more efficient I'll be keeping the existing one around for testing purposes so no future improvements to PDPRecordKeeper will impact tests